pkg/controllers/jivavolume_controller.go

```diff
@@ -199,6 +212,141 @@ func (r *JivaVolumeReconciler) isScaleup(cr *openebsiov1alpha1.JivaVolume) bool
 	return false
 }
 
+func shouldMoveReplicas(cr *openebsiov1alpha1.JivaVolume) bool {
```

Comment:
nit: can we add some functional level comments about the decision we are making here